src/TerminalFighter/MissileLauncher/MissileLauncher.cpp

```diff
@@ -4,12 +4,30 @@
 #include "Missile/Missile.h"
 #include "Util/Util.h"
 
-MissileLauncher::MissileLauncher(Team::Team team, I_GameObjectMediator& game_object_mediator)
-    : team_(team)
+MissileLauncher::MissileLauncher(
+    Team::Team team,
+    I_GameObjectMediator& game_object_mediator,
+    std::string missile_graphic_path,
+    std::string terminal_graphic_path,
+    std::string terminal_font_path,
+    std::string targeting_system_font_path,
+    std::string targeting_system_dict_path,
```

Comment:
Just curious, where do we draw the line on who gets passed "const I_Settings& settings" and who gets strings?

Reply:
Pass in as little as you can, general programming rule. Feels different for settings, but doesn't need to be.